src/integTest/groovy/net/wasdev/wlp/gradle/plugins/LibertyTest.groovy

```diff
@@ -124,4 +124,22 @@ class LibertyTest extends AbstractIntegrationTest{
            throw new AssertionError ("Fail on task Clean. "+e)
         }
     }
+
+    @Test
+    public void test10_run() {
+        try{
+            def stop_thread = Thread.start {
+                sleep(30000)    // sleep 30 sec
```

Comment:
Faced with a 30 second wait, could this thread check the status every 5 seconds and issue the stop after it detects the server is running?  Then after maybe 30 seconds fail the test if the status never shows running?